### PR TITLE
Notes/GitHub and markdown Branch

### PR DESCRIPTION
**Pull request by leonel-acosta**
Branches Project Notes Branch

### CODE REVIEW

File: notes.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+### notes on branches and PR
```
Comment (jasp-de) [inline notes.md:1]: very informative!

**Review by jasp-de — COMMENTED** [review]
nice

**Review by jasp-de — COMMENTED** [review]
well done

**Review by jasp-de — APPROVED** [review]
looks great, well done

File: notes.md (hunk 1 above)
Comment (Freddie2024) [reply notes.md:1]: Yes, indeed!

File: notes.md
Comment (Freddie2024) [inline notes.md:1]: Another lesson in Spanish for me, thank you :)

File: img/git-schema.png
Comment (Freddie2024) [inline img/git-schema.png:1]: I see, this is what we are supposed to do! Nice!

**Review by cc-co-code — COMMENTED** [review]
very good!

**Pull request merged**